src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,99 @@
 package com.theironyard.novauc;
 
+
+import jdk.nashorn.internal.ir.WhileNode;
+
+import java.util.Scanner;
+
+
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
+
+    public static void main(String[] args) throws Exception {
+        Scanner scanner = new Scanner(System.in);
+        int balance = 100;
+        int withdraw = 100-50;
+        double take = 50;
+
+
+        System.out.println("Hello, how are you? Please enter your name.");
+        String name = scanner.nextLine();
+
+        if (name.isEmpty()) {
+            throw new Exception("Ask for name");
+        }
+            System.out.println("Please select following options. 1.balance 2.Withdrawl funds 3 cancel");
+            String options = scanner.nextLine();
+
+        switch (options){
+            case "1":
+             System.out.println("Your balance " + balance);
+             break;
+            case  "2":
+             System.out.println(("How much would you like to withdraw?"));
+           String money = scanner.nextLine();
+             int withdrawl = Integer.valueOf(money);
+             if(withdrawl > balance){
+                 throw new Exception("Insufficent Funds");
+             } else{
+                 System.out.println("Thank you, please take your money ");
+             }
+
+             System.out.println("Withdraw money " + withdraw);
+              System.out.println("Your new balance is " + take );
+              break;
```

Comment:
All in all this is good.  On line 35 you capture how much they money they wanted to take out.  But you never use that variable.  On line 42 you could use it.  So instead of withdraw use withdrawl.  And then after you could do the math on their balance.  Example balance = balance - withdrawl.  Then line 43 use balance instead of take.  Complete and satisfactory, on to the next one.